src/Resources/Locales/en.json

```diff
@@ -583,6 +583,61 @@
             "libraryName": "Panel",
             "description": "A layout panel"
         },
+        "dataHistory": {
+            "title": "Data history widget",
+            "libraryName": "Data history",
+            "description": "Show timeseries data in context of your 3D scene",
+            "notEnabledDescription": "Data history is not enabled for this ADT instance.",
+            "form": {
+                "connectionStringTitle": "Connection string",
+                "connectionStringPlaceholder": "Enter connection string (e.g. key1=value1;key2=value2;key3=value3)",
+                "connectionStringInformation": "Data history widgets target the Azure Data Explorer instance connected to the Azure Digital Twins instance at the time of creation.",
+                "connectionLoadingText": "Loading connection information...",
+                "noConnectionInformationText": "No connection information found",
+                "displayNamePlaceholder": "Enter display name",
+                "timeSeries": {
+                    "timeSeriesDescription": "Add up to {{maxTimeSeriesCount}} timeseries to display",
+                    "title": "Time series",
+                    "description": "A series to display in the line chart",
+                    "property": "Property",
+                    "unit": "Unit",
+                    "label": "Label",
+                    "labelPlaceholder": "Enter label for this series to show in chart",
+                    "add": "Add time series",
+                    "edit": "Edit time series",
+                    "remove": "Remove time series"
+                },
+                "chartOptions": {
+                    "yAxisType": {
+                        "description": "Chart option to set if each series has separate (Independent) y-axes or sharing the same y-axis (Shared)",
```

Comment:
spelling: y-axis

Reply:
it was intentional referring plural axes. is it wrong?

Reply:
i guess you're right. I have never seen it in plural form before. lol I guess it doesn't really come up much.